### PR TITLE
Changes to Clearing Operations

### PR DESCRIPTION
**Pull request by topowright**
Changed folder name to Clearing Operations tools

### CODE REVIEW

**Comment by topowright** [comment]
The last two commits are broken into Arcmap and then ArcGIS Pro.

File: utils/test/clearing_operations_tests/ClearingOperationsCanvasAreaGRGTestCase.py [hunk 1]
```diff
@@ -88,9 +88,9 @@ def testClearingOperationsAreaGRG(self):
         Configuration.Logger.info(runToolMsg)
 
         try:
-			# Calling the CanvasAreaGRGScript_ClearingOperations Script Tool
-            #arcpy.CanvasAreaGRG_ClearingOperations(self.inputArea, cellWidth, cellHeight, cellunits, labelStart, labelStyle, output)
-			arcpy.CanvasAreaGRGScript_ClearingOperations(self.inputArea, cellWidth, cellHeight, cellunits, labelStart, labelStyle, output)
+			# Calling the CanvasAreaGRGS_ClearingOperations Script Tool
```
Comment (mfunk) [inline utils/test/clearing_operations_tests/ClearingOperationsCanvasAreaGRGTestCase.py]: CanvasAreaGRGS (has an extra S) or CanvasAreaGRG?

Comment (topowright) [inline utils/test/clearing_operations_tests/ClearingOperationsCanvasAreaGRGTestCase.py]: I have made this change. Good catch.

**Comment by topowright** [comment]
Those files should be removed.

**Pull request merged**